lib/repl.js

```diff
@@ -521,7 +515,9 @@ REPLServer.prototype.complete = function(line, callback) {
                 }
               }
             }
-          } catch (e) {}
+          } catch (e) {
+            // ignore
+          }
```

Comment:
eslint disallows the empty block.
IMO, we should add some comments when developers ignore exception intentionally.